typescript/lib/config.ts

```diff
@@ -96,8 +98,11 @@ export class AIConfigRuntime implements AIConfig {
    * @param aiConfigFilePath File path to the AIConfig to load.
    */
   public static load(aiConfigFilePath: string) {
-    const aiConfigString = fs.readFileSync(aiConfigFilePath, "utf8");
-    const aiConfigObj = JSON.parse(aiConfigString);
+    let aiConfigObj = yamlToJson(aiConfigFilePath);
+    if (aiConfigObj == null) {
+      const aiConfigString = fs.readFileSync(aiConfigFilePath, "utf8");
```

Comment:
just a nit, but if the .yaml contents are not valid yaml the file ends up being read synchronously twice. Could potentially just read to string once and try parsing as yaml first based on extension